README.md

```diff
@@ -670,13 +679,15 @@ We have several possibilities/tools to optimize the size and speed of our applic
 
 Vue allows us to also create `functional components`, which are basically just stateless functions and meant for a single render. Therefore the rendering itself is much faster while no components instance is cluttering the browser cache.
 
-You can read more about this practice [here](https://vuejs.org/v2/guide/render-function.html#Functional-Components).
+You can read more about this practice [here](https://vuejs.org/guide/extras/render-function.html#Functional-Components).
+
 
 ### Async components
```

Comment:
@patric-eberle, can you please confirm if it's okay that I included more details about the Vue's 3 native `defineAsyncComponent` function and explained how Vite manages dynamic imports?

Reply:
@BeneRichi if there is a Vue provided documentation I would recommend to just add the link. But yeah, that's fine with me. Things changed a lot since this readme was written...